shopsystem-service/src/main/java/com/yandex/market/shopservice/controllers/ShopSystemController.java

```diff
@@ -0,0 +1,56 @@
+package com.yandex.market.shopservice.controllers;
+
+import com.yandex.market.shopservice.dto.ShopSystemDto;
+import com.yandex.market.shopservice.model.shop.ShopSystem;
+import com.yandex.market.shopservice.service.ShopSystemService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/public/api/v1/shopsystems")
+public class ShopSystemController {
+    private final ShopSystemService shopService;
+    private final String REQUEST_NAME = "/public/api/v1/shopsystems";
+
+    @GetMapping()
+    @ResponseStatus(HttpStatus.OK)
+    public Page<ShopSystemDto> getAllShopSystems(@PageableDefault(size = 20) Pageable pageable) {
+        log.info("GET " + REQUEST_NAME +
+                ". Get request received. Pageable parameters: " + pageable);
```

Comment:
Received a request to get paginated list of shop systems. page = ..., size = ....